### ISSUE TITLE
[Feature Request] - LAN sync for clients

### ISSUE THREAD

**Issue by elpoolet** [nextcloud/server#25149]
Hi,
Is there a way to add a LAN sync feature like "DropBox".
I'll explain : 
when you have multiple clients in the same LAN ie: a HTPC wich stores a backup of ALL your photos, each person in the house with his own laptop (my wife, my sons, and I..), and each his smartphone... 
Each smartphone has the Nextcloud app qith auto upload of photos activated.
So, when you take a picture on a phone :  
 - the shared content is uploaded to the nextcloud server (ok it's madatory)
 - then, each computer with sync activated downloads from the server the content, 

If a LAN sync is added, the content would probably be downloaded only once, and if it's very well thinked, not downloaded at all, since the content is already on a device in the LAN (the smartphone, connected by WiFi).

And when you share documents from a laptop, it would be possible to send the content to the server AND to the other devices in the LAN.

I know that nowadays bandwith is not a problem for many people, but in my case, my upload bandwith is VERY POOR (1Mbps in best cases) and my download bandwith is fluctuating between 25Mps and 1Mbps depending the day time...

**Comment by PVince81** [comment]
closing in favor of the client ticket https://github.com/nextcloud/desktop/issues/2010
